homeassistant/components/xiaomi_miio/switch.py

```diff
@@ -251,12 +342,150 @@ async def async_service_handler(service):
                 await asyncio.wait(update_tasks)
 
         for plug_service in SERVICE_TO_METHOD:
-            schema = SERVICE_TO_METHOD[plug_service].get("schema", SERVICE_SCHEMA)
-            hass.services.async_register(
-                DOMAIN, plug_service, async_service_handler, schema=schema
-            )
+            if "schema" in plug_service:
+                schema = SERVICE_TO_METHOD[plug_service].get("schema", SERVICE_SCHEMA)
+                hass.services.async_register(
+                    DOMAIN, plug_service, async_service_handler, schema=schema
+                )
```

Comment:
I cannot commit to this branch directly, all PR's in this branch were already approved. Please create a new PR against the rewok branch to request the changes you want.

Reply:
> 
> 
> What's the reason for this change? When does it happen that there is no schema in `plug_service`?

Yes, because we no not touch the exiting code, only change this part for humidifiers, that doe not use this.